### ISSUE TITLE
tzlocal produces dependency conflict when referencing maya in setup.py

### ISSUE THREAD

**Issue by DmitryDeryabin** [kennethreitz/maya#165]
My `setup.py` looks as follows:

```python
from setuptools import setup

setup(
      ...
      packages=['my-package-name'],
      dependency_links=[
          'git+ssh://private-repo-for-my-package-name'],
      setup_requires=['pytest-runner'],
      install_requires=['maya', 'my-package-name==0.0.1', ...],
      tests_require=['pytest', 'pytest-env', 'pytest-datadir', ...],
      classifiers=[
        "Programming Language :: Python",
        "Programming Language :: Python :: 3",
        "Programming Language :: Python :: 3.6"
      ]
)
```

And when I try to install it via `python setup.py install`, this throws the following error:

`error: tzlocal 2.0.0b1 is installed but tzlocal<2.0.0.0,>=1.5.0.0 is required by {'pendulum'}`

As `pipenv graph` showed me, tzlocal is only required by maya and its dependencies. 

`pip` (and subsequently `pipenv`) seem to be a bit smarter, because running 

```fish
env PIP_PROCESS_DEPENDENCY_LINKS=1 pipenv install -e .
``` 

resolves the dependencies correctly and installs `tzlocal-1.5.1`. 

So, I guess it might help if `tzlocal` version is pinned to 1.5.1 in setup.py for `maya` until `pendulum` version is re-pinned to a newer version.

**Comment by timofurrer** [comment]
I've re-pinned maya in #166 . Can you try if you've still encounter this issue with maya 0.6.0a1 ?

Thanks 🎉

**Comment by timofurrer** [comment]
I'm closing this as not-reproducible. I've released maya v0.6.0

Please try again and come back if the issue still exists :tada:
